### DISCUSSION TITLE
How do I download all the possible audios of a series?

### DISCUSSION THREAD

**Discussion by CrushedAsian255** [crunchy-labs/crunchy-cli#383] [Q&A] [answered]
Instead of passing `-a` for every audio locale, is there a way for crunchy-cli to auto-download all the possible audios (and associated video/subs/whatever)?

**Comment by CrushedAsian255** [comment]
I'm currently just sticking `-a ar-ME -a ar-SA -a ca-ES -a de-DE -a en-IN -a en-US -a es-419 -a es-ES -a es-LA -a fr-FR -a hi-IN -a id-ID -a it-IT -a ja-JP -a ko-KR -a ms-MY -a pl-PL -a pt-BR -a pt-PT -a ru-RU -a ta-IN -a te-IN -a th-TH -a tr-TR -a vi-VN -a zh-CN -a zh-CN -a zh-TW` to the end of my commands to try to get all possible audios

**Comment by bytedream** [comment, marked as answer]
`-a all` will do the job
